cirq/circuits/text_diagram_drawer.py

```diff
@@ -37,15 +37,22 @@ class TextDiagramDrawer:
 
     def __init__(self):
         self.entries = dict()  # type: Dict[Tuple[int, int], str]
+        self.transposed_entries = dict()  # type: Dict[Tuple[int, int], str]
```

Comment:
Instead of adding a second dictionary, change the `entries` dictionary to have type `Dict[Tuple[int, int], Tuple[str, str]]`. Then, instead of using `Tuple[str, str]`, use a `NamedTuple`.

This will come in very handy if we add more entry value types.